namespace-provisioner-gitops-examples/README.md

```diff
@@ -0,0 +1,46 @@
+# Namespace Provisioner Resources
+
+## Namespace provisioner
+
+Namespace provisioner for Tanzu Application Platform provides an easy, secure, automated way for Platform Operators to provision namespaces with the resources and proper namespace-level privileges required for their workloads to function as intended.
+
+
+## Resources
+
+Here you can find examples for extend and for customize the namespace-provisioner default resources
+
+### Extend the OOTB default-resources
+
+Platform Operators may need to do additional namespace customization beyond TAP requirements (such as quota allocation or creation of other namespaced resources) that are not taken care of by the Out of the Box *default-resources* Secret. Platform Operators can add additional git sources in *tap-values* configuration of the namespace provisioner. This allows the Platform Operators to extend the out of the box setup with additional resources which helps them achieve the bespoke customization that they need for their requirement.
+
+#### Custom resources
+
+- Random:  how to add a particular `ConfigMap` using Carvel-YTT *yaml* and *data* modules
+- Scan policies: how to add a `scanpolicy.scanning.apps.tanzu.vmware.com` for *snyk* if the `data.values` contains the key `scanpolicy` and the value is snyk`
+- Tekton pipelines: how to create `pipeline.tekton.dev` for `angular`, `python` or `golang` if the `data.values` contains the key `language` and the value is the desire language
+- Testing scanning supplychain: how to create the `scanpolicy.scanning.apps.tanzu.vmware.com/v1beta1` and the `pipeline.tekton.dev` for java (gradle/maven) projects as referenced in [Install OOTB Supply Chain with Testing and Scanning](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/GUID-getting-started-add-test-and-security.html#install-ootb-supply-chain-with-testing-and-scanning-5)
```

Comment:
since we have an actual TAP version specific reference here, we'll need to update this in the future for tap 1.5, 1.6 etc... when they tag this repo as they do for each TAP release

cc @trisberg -- is there any way to automatically set version values via variable or something?